### PR TITLE
Add DMN resource type support to kie-maven-plugin

### PR DESCRIPTION
**Pull request by tarilabs**

### CODE REVIEW

**Comment by tarilabs** [comment]
@etirelli can you review this one for me, please?

File: kie-maven-plugin/src/test/projects/kjar-2-all-resources/src/main/resources/AllResourceTypes/simple-dmn.dmn [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		<informationRequirement>
+			<requiredInput href="#i_FullName"/>
+		</informationRequirement>
+		<literalExpression>
+			<text>"Hello " + Full Name</text>
+		</literalExpression>
+	</decision>
+	<inputData name="Full Name" id="i_FullName">
+		<variable name="Full Name" typeRef="feel:string"/>
+	</inputData>
```
Comment (psiroky) [inline kie-maven-plugin/src/test/projects/kjar-2-all-resources/src/main/resources/AllResourceTypes/simple-dmn.dmn]: Nit-picking - this file uses tabs, spaces would be preferable.

Comment (tarilabs) [inline kie-maven-plugin/src/test/projects/kjar-2-all-resources/src/main/resources/AllResourceTypes/simple-dmn.dmn]: :+1: seems my Eclipse is doing the TAB->SPACE substitution only for certain files.. updated indentation to suit.

**Comment by etirelli** [comment]
+1

**Pull request merged**